.gitignore

```diff
@@ -10,4 +10,3 @@ npm-debug.log
 # optional dev config file and plugins directory
 .hyper.js
 .hyper_plugins
-
```

Comment:
actually it was the other way around, the new line was missing in the repo
my IDE inserted it according to the .editorconfig